.github/workflows/ios-adhoc-export.yml

```
@@ -0,0 +1,205 @@
+          name: monGARS-AdHoc-IPA
+          path: export/*.ipa
+
+      - name: Cleanup keychain
+        if: always()
+        run: |
+          set -euo pipefail
+          rm -rf signing export exportOptions.plist || true
+          if [ -n "${ORIG_KEYCHAINS_B64:-}" ]; then
+            RESTORE_ARGS=()
+            if ORIG_KEYCHAINS_DECODED=$(python3 - <<'PY'
+import base64, os, sys
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 179: could not find expected ':'

(syntax-check)